keyboards/keebio/sinc/rev3/keymaps/vial/rules.mk

```
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+LTO_ENABLE = yes
+
+ENCODER_MAP_ENABLE = yes
```

```suggestion
ENCODER_MAP_ENABLE = yes

VIALRGB_ENABLE = yes
```
If you've got RGB_MATRIX and plenty of RAM, you might as well leverage Vial's UI.

---

Done.